packages/rs-drive/src/drive/contract/mod.rs

```diff
@@ -868,7 +898,7 @@ impl Drive {
             Ok(Element::Item(stored_contract_bytes, element_flag)) => {
                 let contract = cost_return_on_error_no_add!(
                     &cost,
-                    <Contract as DriveContractExt>::from_cbor(&stored_contract_bytes, None,)
+                    DataContract::deserialize_no_limit(&stored_contract_bytes)
```

Comment:
I was wondering why it's useful to do deserilaize no limit? Like I see it used here, but can we actually store a contract that is larger than the limit?